### ISSUE TITLE
(website playground) panicked at 'range end index 12 out of range for slice of length 9'

### ISSUE THREAD

**Issue by qexat** [uiua-lang/uiua#35]
Hi, it's me again :sweat_smile:. It seems that I found another runtime error x)

## Program

```
func ← ↯[⧻∶⧻]..
func [3 5 2]

⌂≡(⊗).
```

## Expected output

Not sure, I wanted to try `grades` to understand what it did

## Actual output

None. "Running" blinks forever.

## Console

Full traceback in the next comment.

## Specs

Browser: Google Chrome 118.0.5993.32 (Official Build) beta (64-bit)

**Comment by qexat** [comment]
I truncated the errors related to extensions or that are not relevant to this issue.

```
[truncated]

site-11b5530f73ff5e21.js:384 panicked at 'range end index 12 out of range for slice of length 9', src\algorithm\monadic.rs:253:18

Stack:

Error
    at imports.wbg.__wbg_new_abda76e883ba8a5f (https://www.uiua.org/site-11b5530f73ff5e21.js:371:21)
    at console_error_panic_hook::hook::h51324b83e5e34372 (https://www.uiua.org/site-11b5530f73ff5e21_bg.wasm:wasm-function[1407]:0x51c825)
    at std::panicking::rust_panic_with_hook::h7601402c0a383194 (https://www.uiua.org/site-11b5530f73ff5e21_bg.wasm:wasm-function[2877]:0x5b10e3)
    at std::panicking::begin_panic_handler::{{closure}}::he405aaeb801d5772 (https://www.uiua.org/site-11b5530f73ff5e21_bg.wasm:wasm-function[3070]:0x5b9a25)
    at std::sys_common::backtrace::__rust_end_short_backtrace::h04fac26f88d230df (https://www.uiua.org/site-11b5530f73ff5e21_bg.wasm:wasm-function[4139]:0x5ccf0b)
    at rust_begin_unwind (https://www.uiua.org/site-11b5530f73ff5e21_bg.wasm:wasm-function[3666]:0x5c90b8)
    at core::panicking::panic_fmt::h6dad0405f48e39e2 (https://www.uiua.org/site-11b5530f73ff5e21_bg.wasm:wasm-function[3701]:0x5c9974)
    at core::slice::index::slice_end_index_len_fail::he06aaf6517598049 (https://www.uiua.org/site-11b5530f73ff5e21_bg.wasm:wasm-function[3241]:0x5bfb50)
    at uiua::array::Array<T>::row_slice::hb7eb1e5e3372bc08 (https://www.uiua.org/site-11b5530f73ff5e21_bg.wasm:wasm-function[1919]:0x569f84)
    at core::slice::sort::insertion_sort_shift_left::h7c196775b5673764 (https://www.uiua.org/site-11b5530f73ff5e21_bg.wasm:wasm-function[1701]:0x54eb24)


imports.wbg.__wbg_error_f851667af71bcfc6 @ site-11b5530f73ff5e21.js:384
site-11b5530f73ff5e21_bg.wasm:0x5ccfeb Uncaught RuntimeError: unreachable
    at rust_panic (site-11b5530f73ff5e21_bg.wasm:0x5ccfeb)
    at std::panicking::rust_panic_with_hook::h7601402c0a383194 (site-11b5530f73ff5e21_bg.wasm:0x5b110c)
    at std::panicking::begin_panic_handler::{{closure}}::he405aaeb801d5772 (site-11b5530f73ff5e21_bg.wasm:0x5b9a25)
    at std::sys_common::backtrace::__rust_end_short_backtrace::h04fac26f88d230df (site-11b5530f73ff5e21_bg.wasm:0x5ccf0b)
    at rust_begin_unwind (site-11b5530f73ff5e21_bg.wasm:0x5c90b8)
    at core::panicking::panic_fmt::h6dad0405f48e39e2 (site-11b5530f73ff5e21_bg.wasm:0x5c9974)
    at core::slice::index::slice_end_index_len_fail::he06aaf6517598049 (site-11b5530f73ff5e21_bg.wasm:0x5bfb50)
    at uiua::array::Array<T>::row_slice::hb7eb1e5e3372bc08 (site-11b5530f73ff5e21_bg.wasm:0x569f84)
    at core::slice::sort::insertion_sort_shift_left::h7c196775b5673764 (site-11b5530f73ff5e21_bg.wasm:0x54eb24)
    at uiua::algorithm::monadic::<impl uiua::value::Value>::grade::ha53322935b9f0855 (site-11b5530f73ff5e21_bg.wasm:0x1dc232)
invalid/:1 

[truncated]
```

**Comment by kaikalii** [comment]
Thanks for the report.
Fixed in 28aaa73c0cf9
